articles/api-auth/dynamic-client-registration.md

```diff
@@ -0,0 +1,205 @@
+---
+description: How to dynamically register Clients with Auth0, using the Management APIv2.
+toc: true
+---
+
+# Dynamic Client Registration
+
+Dynamic Client Registration is a feature, based on the [OpenID Connect Dynamic Client Registration specification](https://openid.net/specs/openid-connect-registration-1_0.html), that enables you to to register clients dynamically, using the [Management APIv2](/api/management/v2). This way you can allow third party developers to register and use clients.
```

Comment:
Note that while the code lives in the same deployment, we don't consider the dynamic registration endpoint part of API2. API2 endpoints live under `/api/v2/something` while the new endpoint is just `tenant.auth0.com/oidc/register`.